documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,172 @@
+... (Optional) Click `Advanced` to view additional optional advanced configuration for the new Maven project, such as:
+.... `Resolve Workspace projects`: dependencies opened as workspace projects will be resolved without having to install them to your local Maven repository first. This way, any changes made to one of these dependencies will have an immediate effect on other projects consuming it (compilation, refactoring, etc.). When `Resolve Workspace projects` is disabled, dependencies existing in the workspace must be installed to your local Maven repository after any change (by running `mvn install`), in order to see effects in projects consuming them.
+.... `Profiles`: select a set of Maven profiles to activate or deactivate in the workspace. Profiles are defined in the project `pom.xml`, or inherited from a parent `pom.xml`, or defined in the relevant `settings.xml`.
+.... `Name templates`: allows you to disambiguate projects names in the workspace by prepending or appending the group ID or SCM branch names to the default artifact ID.
++
+.Create a New Maven Project
+image::mvnbasic_create_project.png[Create a New Maven Project]
++
+.. When the configuration is complete, click `Next` to continue.
+. In the second step of the `New Maven Project` wizard, configure the project details:
+.. Enter the desired group ID, which is similar to an organization namespace (for example, *com.company.businessunit.project*), in the `Group Id` field.
+.. Enter the desired artifact ID value, which is the name for your project, in the `Artifact Id` field. This value must not include any spaces and the only special characters allowed are periods ('.'), underscores ('_'), and dashes ('-').
+.. Set the `Version` field’s value to `0.0.1-SNAPSHOT` or a similar value. For details about the appropriate version build numbers, see http://books.sonatype.com/mvnref-book/reference/pom-relationships-sect-pom-syntax.html[Project Versions]
+.. Set the value of the `Packaging` field to `pom`.
+.. (Optional) Add a name for your project in the `Name` field.
+.. (Optional) Add a description for your project in the `Description` field.
++
+.Configure Project Details
+image::mvnbasic_maven_project.png[Configure Project Details]
++
+.. Click `Finish` to conclude the new Maven project creation wizard.
+
+*Result:* Your new Maven project is created and appears in the `Project Explorer` view.
+
+[[New_Module]]
+== 2. Create a New Maven Module
+
+Each Maven project with a packaging pom can include multiple Maven modules. Follow the instructions to create your first Maven module:
+
+=== Prerequisites
+. You must have an existing Maven project available with the packaging type *pom*. See <<New_Project,Create a New Maven Project>> for instructions to create a new Maven project.
+
+=== Creating a New Maven Module
+. Create a new Maven module as follows:
+.. In your workspace, navigate to menu:File[New > Other].
```

Comment:
In the project explorer view, right click on your recently created pom project and select New > Project.